.gitignore

```diff
@@ -3,3 +3,6 @@
 /target
 Cargo.lock
 /book/book
+ /docs/_build
+/docs/source/.doctrees
+.idea/workspace.xml
```

Comment:
Remove the ``.idea`` directory from the repository. These files are generated by the IDE.

Then, gitignore the complete folder adding to this file:

```.idea```